### ISSUE TITLE
Editor: Properly convert foreign characters

### ISSUE THREAD

**Issue by Innders** [ynput/ayon-frontend#827]
![image](https://github.com/user-attachments/assets/6abdcfd7-82f6-4047-a0b6-581383a584d9)

**Comment by martastain** [comment]
This is the method we use in the backend https://github.com/immstudios/nxtools/blob/develop/nxtools/text.py#L23

- transliterate characters to ascii
- drop characters that are not considered "word-splitting"
- replace all "splitting" characters with separator (underscore in our case)
- ensure there aren't multiple separators in the row

**Comment by mkolar** [comment]
I don't think this is really crucial right now. We'll need to re-write the editor anyways, so I'd skip improving it right now.
